ui/src/pages/topics/[id]/index.tsx

```diff
@@ -0,0 +1,160 @@
+                            Language:{' '}
+                            {Config.values.octopusInformation.languages.find((l) => l.code === topic.language)?.name}
+                        </p>
+                        {user && user.email ? (
+                            <div className="pb-10">
+                                <Components.Link
+                                    href={`${Config.urls.createPublication.path}?topic=${topic.id}&type=PROBLEM`}
+                                    className="flex w-fit text-teal-600 dark:text-teal-400"
+                                >
+                                    Write a linked Research Problem
+                                    <OutlineIcons.PencilIcon className="ml-2 h-5 w-5" />
+                                </Components.Link>
+                            </div>
+                        ) : user && !user.email ? (
+                            <div className="pb-10">
+                                <Components.Link
+                                    href={`${Config.urls.verify.path}?state=${encodeURIComponent(
+                                        `${Config.urls.viewTopic.path}/${topic.id}`
+                                    )}`}
+                                    className="flex w-fit items-center rounded border-transparent text-sm font-medium text-teal-600 outline-0 transition-colors duration-500 hover:underline focus:overflow-hidden focus:ring-2 focus:ring-yellow-400 dark:text-teal-400"
+                                >
+                                    Verify your email for more actions
+                                </Components.Link>
+                            </div>
+                        ) : (
+                            <div className="pb-10">
+                                <Components.Link
+                                    href={`${Config.urls.orcidLogin.path}&state=${encodeURIComponent(
+                                        `${Config.urls.viewTopic.path}/${topic.id}`
+                                    )}`}
+                                    className="flex w-fit items-center rounded border-transparent text-sm font-medium text-teal-600 outline-0 transition-colors duration-500 hover:underline focus:overflow-hidden focus:ring-2 focus:ring-yellow-400 dark:text-teal-400"
+                                >
+                                    <Assets.ORCID
+                                        width={25}
+                                        height={25}
+                                        className="mr-2 rounded-md bg-orcid fill-white-50 p-1"
+                                    />
+                                    <span> Sign in for more actions</span>
+                                </Components.Link>
+                            </div>
+                        )}
+                    </header>
+                    <p className="border-b border-grey-200 py-10">{Config.values.topicDescription}</p>
```

Comment:
you can take out the border bottom from the description and put a border top on each of the sections below description, so that the border appears conditionally above each section and not to the end of them.

![image](https://github.com/JiscSD/octopus/assets/48569671/6bf7be75-a752-4d6b-b64a-c5ed1691d6f6)

Reply:
Good idea, I've just added this change